content/en/device-support/_index.md

```diff
@@ -228,4 +251,5 @@ Passkeys created in **macOS** can be used on:
     <br><sup>2</sup> See <a href="/docs/reference/macos/#browser-behavior" target="_blank">macOS browser
         behavior</a> for caveats
     <br><sup>3</sup> Chrome M108 and Windows 11 22H2
+    <br><sup>4</sup> Android: requires <a href="https://developers.google.com/identity/passkeys/supported-environments#android-passkey-support">Android 9+ and the Credential Manager Alpha</a>; iOS/macOS: see <a href="https://developer.apple.com/documentation/authenticationservices/public-private_key_authentication/supporting_passkeys">Apple Developer Documentation</a>
```

Comment:
IMO, this is too much for a superscript list. We can add these details under the reference pages.

Reply:
Sure thing, will move these out then 👍